spec/support/cuprite/domain_restricted_driver.rb

```diff
@@ -0,0 +1,19 @@
+module Cuprite::DomainRestrictedDriver
```

Comment:
It would maybe be somewhat better to have just had the `Capybara.register_driver(:domain_restricted_cuprite)` call at the top level of this file, but Zeitwerk complains about this. Putting it as a static module method (where the module name matches the file path) satisfies Zeitwerk.